specification/gedcom-3-structures-3-meaning.md

```diff
@@ -1572,9 +1567,21 @@ Other descriptor values might include, for example,
 See also `FACT` and `EVEN` for additional examples.
 :::
 
-#### `TYPE` (Type) `g7:NAME-TYPE`
+#### `TYPE` (Type) `g8:NAME-TYPE`
+
+An list of enumerated values from set `g8:enumset-NAME-TYPE` indicating the types of the name.
+The order of values in the list is not significant.
```

Comment:
Discussion For 8.0, we should consider deleting the List-Enum construct alltogether since merging list-enum payloads requires a custom algorithm whereas using multiple Enum structures uses the generic merging algorithm and so is less bug prone.